src/app/hooks/useMediaQuery/index.jsx

```diff
@@ -6,8 +6,8 @@ const useMediaQuery = (query, handler) => {
 
     handler(mediaQueryList);
 
-    mediaQueryList.addListener(handler);
-    return () => mediaQueryList.removeListener(handler);
+    mediaQueryList.addEventListener('change', handler);
```

Comment:
Given the existing method seems to be deprecated, but the new method is also not as widely supported, I think we should add a fallback here.

```suggestion
if (mediaQueryList.addEventListener) {
  mediaQueryList.addEventListener('change', handler);
} else {
  mediaQueryList.addListener(handler);
}
```